diff -pruN 3.6-14/src/recodext.h 3.6-14ubuntu1/src/recodext.h
--- 3.6-14/src/recodext.h	2001-01-04 14:36:54.000000000 +0000
+++ 3.6-14ubuntu1/src/recodext.h	2008-01-24 20:11:52.000000000 +0000
@@ -218,7 +218,7 @@ struct recode_symbol
     enum recode_symbol_type type : 3;
 
     /* Non zero if this one should be ignored.  */
-    bool ignore : 2;
+    bool ignore : 1;
   };
 
 struct recode_surface_list
