--- iconfig.cpp.orig	2009-10-12 17:10:55.000000000 +0200
+++ iconfig.cpp	2009-10-12 17:11:15.000000000 +0200
@@ -34,6 +34,8 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
+#include <stdlib.h>
+
 
 
 template <class T>

--- wm.cpp.orig	2009-10-12 17:12:09.000000000 +0200
+++ wm.cpp	2009-10-12 17:12:21.000000000 +0200
@@ -28,6 +28,7 @@
 #include <time.h>
 #include "atoms.h"
 
+#include <stdlib.h>
 
 WM * WM::m_instance = 0;
 bool WM::x_error = false;
